CONTRIBUTING.md

```diff
@@ -61,7 +61,12 @@ If you wish to build a copy of Babel for distribution, then run:
 $ make build-dist
 ```
 
-and access the files from `packages/babel-core/dist`.
+and access the built files for individual packages from `packages/<package-name>/lib`.
+
+The "top-level" package is `babel-cli`.  Accordingly, its built files live in `packages/babel-cli/lib`.
```

Comment:
Top-level surely means what is above everything else intra-codebase, not intra-codebase. Still a note to the effect that it is also common practice for people use `babel-core` indirectly through other modules would be welcome.
Obviously I'm a newbie and Babel members have much better insight. Nevertheless, I spent way too much time trying to follow the existing instruction which was just plain wrong. So my first contribution was an attempt to improve the contributing guide so that others would not waste time like I did. 

Please feel free to improve my attempted improvements with your greater level of knowledge.